exercises/list-ops/example.rb

```diff
@@ -0,0 +1,67 @@
+class ListOps
```

Comment:
I lean slightly towards the direction of preferring the real names, because people already know what they do. It's obvious when looking at the code that these aren't just using the standard library implementations, so I think that would be totally fine.